addons/mail/static/src/activity/activity_list_popover.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+    <t t-name="mail.ActivityListPopover" owl="1">
+        <div class="o-activity-list-popover d-flex flex-column" t-ref="root">
+            <div class="o-activity-list-popover-core d-flex flex-column flex-grow-1">
+                <t t-if="state.activities.length === 0">
+                    <span class="p-3 text-center fst-italic text-500 border-bottom">Schedule activities to help you get things done.</span>
+                </t>
+                <t t-if="overdueActivities.length > 0">
+                    <div class="d-flex bg-100 py-2 border-bottom">
+                        <span class="o-activity-list-popover-overdue-title text-danger fw-bold mx-3">Overdue</span>
+                        <span class="flex-grow-1"/>
+                        <span class="badge rounded-pill text-bg-danger mx-3 align-self-center" t-esc="overdueActivities.length"/>
+                    </div>
+                    <t t-foreach="overdueActivities" t-as="activity" t-key="activity.id">
+                        <ActivityListPopoverItem activity="activity" onActivityChanged="props.onActivityChanged" onClickEditActivityButton="props.close"/>
+                    </t>
+                </t>
+                <t t-if="todayActivities.length > 0">
+                    <div class="d-flex bg-100 py-2 border-bottom">
+                        <span class="o-activity-list-popover-today-title text-warning fw-bold mx-3">Today</span>
+                        <span class="flex-grow-1"/>
+                        <span class="badge rounded-pill text-bg-warning mx-3 align-self-center" t-esc="todayActivities.length"/>
+                    </div>
+                    <t t-foreach="todayActivities" t-as="activity" t-key="activity.id">
+                        <ActivityListPopoverItem activity="activity" onActivityChanged="props.onActivityChanged" onClickEditActivityButton="props.close"/>
+                    </t>
+                </t>
+                <t t-if="plannedActivities.length > 0">
+                    <div class="d-flex bg-100 py-2 border-bottom">
+                        <span class="o-activity-list-popover-planned-title text-success fw-bold mx-3">Planned</span>
+                        <span class="flex-grow-1"/>
+                        <span class="badge rounded-pill text-bg-success mx-3 align-self-center" t-esc="plannedActivities.length"/>
+                    </div>
+                    <t t-foreach="plannedActivities" t-as="activity" t-key="activity.id">
+                        <ActivityListPopoverItem activity="activity" onActivityChanged="props.onActivityChanged" onClickEditActivityButton="props.close"/>
+                    </t>
```

Comment:
Can use `t-if` and `t-foreach` on single child (no all of them, but majority of them in this diff).

Reply:
This was just a move of the template present in the old folder, I didn't change the structure here.

Maybe if the old folder is actually unsync with code from master due to rebase issue we should resolve that first?